hamlpy/parser/elements.py

```diff
@@ -190,10 +191,10 @@ def attr_wrap(val):
 
         return " ".join(rendered)
 
-    @staticmethod
-    def _escape_attribute_quotes(v, attr_wrapper):
+    @classmethod
```

Comment:
we need to update https://github.com/nyaruka/django-hamlpy/pull/96/files#diff-971f769c231cfc0bbf4b9a88f9eff1ec4b736a8ed724a1402f1ad7c26c6f8181R183 ?

Reply:
And probably update tests that really use `render_attributes`?

Reply:
I'm not following... why do we need to update that line?

Reply:
I was asking to use the class instead of the instance self?

Reply:
ok